sdcm/logcollector.py

```diff
@@ -1222,6 +1222,35 @@ def get_gce_instances_by_testid(self):
                                                       instance=instance,
                                                       global_ip=instance.public_ips[0],
                                                       tags={**self.tags, "NodeType": "loader", }))
+        if self.params["use_cloud_manager"]:
+            try:
+                from cluster_cloud import \
+                    get_gce_manager_instance_by_cluster_id  # pylint: disable=import-outside-toplevel
+            except ImportError:
+                LOGGER.error("Couldn't collect Siren manager logs, cluster_cloud module isn't installed")
+            else:
+                cloud_manager_id = self.params["cloud_manager_id"] if "cloud_manager_id" in self.params else None
```

Comment:
It easier just to use .get, self.params is a dictionary.
```suggestion
                cloud_manager_id = self.params.get("cloud_manager_id")
```

Reply:
"cloud_manager_id" is not defined config param. I don;t think we need to add it as parameter and set default value

Reply:
as in any dict, we don't need it defined. `dict.get` would return None if the key doesn't exist.

Reply:
Changed. I didn't run the test